wormhole-connect/src/views/v2/Bridge/AssetPicker/TokenItem.tsx

```diff
@@ -53,7 +54,9 @@ function TokenItem(props: TokenItemProps) {
   const address =
     chain === token.nativeChain
       ? token.tokenId?.address
-      : getWrappedToken(token)?.foreignAssets?.[chain];
```

Comment:
so we're no longer using `getWrappedToken` which comes from `utils`... but I feel like it would be a good idea to maintain that method or one like it and hiding the implementation detail of `config.wrappedTokenAddressCache` existing.

we have [`getTokenBridgeWrappedTokenAddress`](https://github.com/wormhole-foundation/wormhole-connect/blob/development/wormhole-connect/src/utils/sdkv2.ts#L65) in `utils/sdkv2.ts` but it's async... maybe we could add one like it that's synchronous and only hits this cache.

I just think it would be nice to have a util for this and not have to reference this cache (again, an implementation detail) in our React code like this

Reply:
agree that we should hide impl details